scripts/scripts.js

```diff
@@ -408,6 +417,7 @@ export function getConsent(topic) {
 
 async function loadPage() {
   await loadEager(document);
+  listenForLCPEvent();
```

Comment:
I think this should just be in loadEager. But not going to hold the PR on it.

Reply:
I had it there before. In my opinion, here it makes the page-loading sequence more evident.